_data/sites.json

```
@@ -9478,6 +9478,15 @@
             "shopify.com"
         ]
     },
+    {
```

```suggestion

    {
```
I meant to suggest just removing the trailing whitespaces, but not the line break, sorry.